EpiAware/docs/src/showcase/replications/mishra-2020/index.jl

```diff
@@ -76,6 +96,28 @@ C_t \sim \text{NegBin}(\text{mean} = I_t,~ \text{overdispersion} = \phi).
 In the examples below we are going to largely recreate the _Mishra et al_ model, whilst emphasing that each component of the overall epidemiological model is, itself, a stand alone model that can be sampled from.
 "
 
+# ╔═╡ 1d3b9541-80ad-41b5-a5ed-a947f5c0731b
+md"
+## Load the data into scope
+First, we make sure that we have the data we want to analysis in scope by reading from a local CSV file. As with the environment, this notebook works relative to its directory. If you are running this from the REPL from the root of the `RtWithoutRenewal` repository then replace the following:
+
+```julia
+notebook_dir = \"EpiAware/docs/src/showcase/replications/mishra-2020\"
```

Comment:
Might want to make this system agnostic (e.g. aren't the slashes the other way in Windows?) In R I always use file.path("each", "component")

Reply:
Thats a good point... I've come up with a different solution where we grab directly from the repo like here https://turinglang.org/TuringGLM.jl/dev/tutorials/linear_regression/ .

I'll push that, and that makes a lot more sense!

Reply:
Yeah I think aiming for copying and pasting we need to be downloading from a link. Sounds like a good change.